src/Mod/Sketcher/Gui/Utils.cpp

```diff
@@ -57,14 +57,17 @@ bool SketcherGui::tryAutoRecompute(Sketcher::SketchObject* obj, bool& autoremove
 
     // We need to make sure the solver has right redundancy information before trying to remove the
     // redundants. for example if a non-driving constraint has been added.
-    if (autoRemoveRedundants && autoRecompute)
```

Comment:
For all these syntax modification it would be best to group them in one separate commit so that they can be reviewed faster. Because as they are all squashed we have to go through them all to check if you actually changed something or just fixed the syntax.
Edit: And there are a lot of them, like 80% of the PR is syntax fixes. Two separates PR would even be best as this makes the PR look very big when it is in fact not so big.

Reply:
Hi @PaddleStroke, syntax change is mainly due to the clang precommit hook. See #9140 and https://github.com/FreeCAD/DevelopersHandbook/blob/master/codeformatting/index.md

Reply:
Ah ok sorry about that I didn't knew. I was thinking you were very meticulous in your file edition 😄